src/Microsoft.AspNetCore.Server.Kestrel.Core/CoreStrings.resx

```diff
@@ -130,7 +130,7 @@
     <value>Chunked request incomplete.</value>
   </data>
   <data name="BadRequest_FinalTransferCodingNotChunked" xml:space="preserve">
-    <value>Final transfer coding is not "chunked": "{detail}"</value>
+    <value>The message body length cannot be determined because the final transfer encoding was set to '{detail}' instead of 'chunked'.</value>
```

Comment:
For whatever reason the spec always refers to this as "transfer coding", despite the header name being called Transfer-**En**coding. That's why it was spelled like that in the original message.